### ISSUE TITLE
Extracting file containing ":" or any symbol not allowed by windows gives errors

### ISSUE THREAD

**Issue by hliriano03** [adamhathcock/sharpcompress#543]
I am trying to extract an .IPA file, iTunes App. This app has a file with ":" in it and crashes program when trying to extract it due to not being allowed to create a folder with such character. I tried finding a work around but to no avail.

**Comment by adamhathcock** [comment]
what code are you using?

You probably have to do a more manual approach for extraction if there are invalid path characters in the archive key.

Don't use `WriteAllToDirectory`

**Comment by hliriano03** [comment]
Yes i am using that code. Due to once im done extracting i would need to compile .ipa file back exactly how it was.
